components/src/hardware-sim/Deck/RobotWorkSpace.tsx

```diff
@@ -33,6 +34,7 @@ export function RobotWorkSpace(props: RobotWorkSpaceProps): JSX.Element | null {
     deckLayerBlocklist = [],
     trashSlotName,
     viewBox,
+    trashColor,
```

Comment:
maybe 
```suggestion
    trashColor = '#757070',
```

Then we can remove optional from trashColor from `StyledDeck.tsx`
trashColor is not used in this component.

Reply:
sure! i can move it up here

Reply:
oh nevermind, i know why i made it optional. `StyledDeck` is also being used in `MoveLabwareOnDeck`, so I think its better to keep it optional and default to `'#757070'` in `StyledDeck`

Reply:
okay